### ISSUE TITLE
[Bug]: The 640x480 resolution is not playable with the current size of the interface elements

### ISSUE THREAD

**Issue by LC-Zorg** [OpenTTD/OpenTTD#9933]
### Version of OpenTTD

12.2

### Expected result

The smallest available resolution is 800x600 or even higher

### Actual result

In the case of using the 640x480 resolution, some interface elements do not fit on the screen and some functions are unavailable.

### Steps to reproduce

1. Change the resolution to 640x480
2. Open the game options window and try changing the base music set or see readme - imposible
3. Add FIRS 4 to the game
4. Open the graphs window and decide what you prefer to see: rates values or cargoes names
5. Change the resolution to 800x600
6. Open the industry directory window - its width, depending on the set used, language and city name, can be even over 1210 pixels
7. Add U& roads and tracks to the game - you will see only part of the list without being able to see the rest

**Comment by PeterN** [comment]
In case of people thinking 640x480 is archaic, note that if the GUI size is 2x then at 1920x1080 (which is fairly common), the effective UI design will need to be for at most 960x540, which is not much more than 480 rows.

**Comment by 2TallTyler** [comment]
I tested this after #10114 and #10264 and found that some windows now fit, while others do not.

Fixed:
* Cargo payment rates, even with FIRS 4 Steeltown, fit just fine
* Road and rail selection dropdowns now have a scrollbar

Still don't fit:
* Options menu. It needs a scrollbar (#10170) but is also too wide for the screen.
* Industry directory is still too wide

**Comment by PeterN** [comment]
* Options windows fits now.
* Industry directory list probably needs a redesign—some industries in FIRS 4 produce 5+ cargo types.

**Comment by 2TallTyler** [comment]
Yes, and I'll add the station directory to the list. Using the AXIS industry GRF and a 2x interface scale, the cargo acronym tabs make it much wider than my monitor.

![station_directory](https://github.com/OpenTTD/OpenTTD/assets/55058389/526bf19b-3019-441f-92e2-1598a040d0dd)

**Comment by PeterN** [comment]
Ah, more to fix! :-)
